content/en/docs/getting-started/_index.md

```diff
@@ -38,6 +38,10 @@ sudo dnf install -y cri-o cri-tools podman
 sudo systemctl enable crio --now
 ```
 
+{{< note >}}
+If the above does not work for your OS, refer to [CRI-O install documentation](https://github.com/cri-o/cri-o/blob/main/install.md).
```

Comment:
We're making explicit above that we expect this to to be run on RHEL/CentOS (Stream)/Fedora only, so I don't think the note for other distros or even OSes is necessary in this case. However, these instructions _should_ then work on CentOS Stream, too, of course.

Can we maybe add the CentOS Stream case above? If this becomes unwieldy, maybe change to a `case` statement or shall we even move this into a tab per distro?

Reply:
sure, sg, i'll add it then remove the note.

Reply:
updated with tabs, ptal